systems/registry/users/pb/users.proto

```diff
@@ -26,15 +26,15 @@ message AddResponse{
 }
 
 message GetRequest {
-    string userUuid = 1 [(validator.field) = {uuid_ver: 0, string_not_empty: true}, json_name = "user_uuid"];
+    string userUuid = 1 [(validator.field) = {uuid_ver: 4, string_not_empty: true}, json_name = "user_uuid"];
 }
 
 message GetResponse {
     User user = 1;
 }
 
 message UpdateRequest {
-    string userUuid = 1 [(validator.field) = {uuid_ver: 0, string_not_empty: true}, json_name = "user_uuid"];
+    string userUuid = 1 [(validator.field) = {uuid_ver: 4, string_not_empty: true}, json_name = "user_uuid"];
```

Comment:
Instead of using `useruuid`, which is already known to be a UUID, let's use 'userID' to make the variable name more concise.

Reply:
Thanks. But there's already a User.ID (because I was using gorm.Model)  which will translate into UserID in DB foreign keys. That's why I've kept the UserUuid.